tsconfig.json

```diff
@@ -9,10 +9,10 @@
     "noUnusedLocals": true,
     "outDir": "dist",
     "removeComments": true,
-    "rootDir": "src/main",
+    "rootDir": "src",
```

Comment:
Let's move test files next to source code files then we might not need to change the `rootDir` here.